project-setup

```diff
@@ -0,0 +1,181 @@
+    "here https://github.com/TheIndexingProject/contratospr-api/blob/master/CONTRIBUTING.md#project-setup. 👀"
+echo "If this doesn't help or is even more confussing for you please let use know " \
+    "by creating a new issue here https://github.com/TheIndexingProject/contratospr-api/issues/new/choose"
+echo ""
+
+read -p "Do you wish to start? [Y/n] " choice
+echo ""
+
+if ! [ $choice == "Y" ] && ! [ $choice == "y" ] ; then
+    echo "Exiting setup. 👋🏼"
+    exit 1
+fi
+
+if ! [ -x "$(command -v docker)" ] ; then
+    echo "🛑 We could not detect if you have Docker installed. " \
+        "We recommend you install both Docker for your system. 🛑"
+    echo "For information on how to do that please take a look at https://docs.docker.com/install/"
+
+    exit 1
+else
+    echo "Found Docker 🐳"
+fi
+
+echo ""
+
+if ! [ -x "$(command -v docker-compose)" ] ; then
+    echo "🛑 We could not detect if you have Docker Compose installed. " \
+        "We recommend you install Docker Compose for your system. 🛑"
+    echo "For information on how to do that please take a look at https://docs.docker.com/compose/install/"
+
+    exit 1
+else
+    echo "Found Docker Compose ✅"
+fi
+
+echo ""
+
+if ! [ -x "$(command -v python3.7)" ] || ! [ -x "$(command -v python3)" ]; then
+    if ! [ -x "$(command -v python)" ]; then
+        echo "🛑 We could not detect that you have Python installed. Please install Python 3.7.x. 🛑"
+        exit 1
+    else
+        python_version=`python -c 'import sys; version=sys.version_info[:3]; print("{0}.{1}".format(*version))'`
+
+        if [ $python_version != "3.7" ] ; then 
+            echo "Please install Python version 3.7.2 or greater"
+            exit 1
+        fi
+    fi
+fi
+
+echo "Found Python 3.7 🐍"
+echo ""
+
+if ! [ -x "$(command -v pipenv)" ]; then
+
+    read -p "You do not have pipenv installed. The default installation method will be Pip. Would you like to install it with [P]ip or [H]omebrew? " choice
+
+    if [ $choice == "h" ] || [ $choice == "H" ] ; then
+        if ! [ -x "$(command -v brew)" ] ; then
+            echo "🛑 We could not find Homebrew installed in your system. " \
+                "Please take a look on how to install it here: https://brew.sh/ 🛑"
+            exit 1
+        fi
+
+        echo "Found Homebre 🍺. Installing Pipenv."
+        brew install pipenv
+    else
+        echo "Installing using Pip"
+
+        if [ -x "$(command -v pip3.7)" ]; then
+            pip3.7 install pipenv
+        elif [ -x "$(command -v pip3)" ]; then
+            pip3 install pipenv
+        else
+            echo "🛑 We could not find pip installed in your system. 🛑"
+            exit 1
+        fi
+    fi
+fi
+
+pipenv install --dev
+
+pipenv run pre-commit install
+
+echo ""
+
+echo "Creating the .env file. Remember to change this file with your settings."
+if [ -f example.env ]; then
+    cp example.env .env
+else
+    echo "PYTHONUNBUFFERED=1" >> .env
```

Comment:
I’d remove this else branch. We’re better off getting rid of the .env for development and move these inside the docker-compose. When we do, will be able to remove this whole part too.

Reply:
Fair point.